baybe/utils/dataframe.py

```diff
@@ -8,6 +8,7 @@
 import numpy as np
 import pandas as pd
 import torch
+from pandas import option_context
```

Comment:
The file already imports `pandas`. It is thus not necessary to re-import a single function from pandas, since you can just use `pd.option_context` at all places where you currently use `option_context`.